modules/mono/build_scripts/mono_configure.py

```diff
@@ -406,7 +406,7 @@ def make_template_dir(env, mono_root):
 
     assert is_desktop(platform)
 
-    template_dir_name = "data.mono.%s.%s.%s" % (platform, env["bits"], target)
+    template_dir_name = "data.mono.%s.%s.%s" % (platform, env["arch"], target)
```

Comment:
This will break C# exporter code in `GodotTools`, which is unchanged in this PR.

Reply:
This are the lines:
https://github.com/godotengine/godot/blob/21b81998bb42b2324a539f01354b68e4fba2c696/modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs#L334-L338

Reply:
I looked around and this would require a lot of work to fix, as it would involve adding a dropdown in ExportPlugin for the architecture so that this information can be passed down to here. I'll see what I can do.

Reply:
Are you sure about that? I think features includes the architectures. See:

https://github.com/godotengine/godot/blob/21b81998bb42b2324a539f01354b68e4fba2c696/modules/mono/editor/GodotTools/GodotTools/Export/AotBuilder.cs#L536-L557

Or is that only added for mobile platforms?

Alternatively, since features still includes the bits (this PR doesn't seem to remove that), you could change that to `features.Contains("64") ? "x86_64" : "x86_32"`.

Reply:
I think we'd want to make this work for any target architecture, so I'm inclined on passing `env["arch"]` all the way down. However it could also be done in a follow-up PR, so in the meantime I changed the above back to `env["bits"]`.